gazebo_msgs/package.xml

```diff
@@ -1,36 +1,36 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>gazebo_msgs</name>
   <version>2.8.4</version>
   <description>
-    Message and service data structures for interacting with Gazebo from ROS.
+    Message and service data structures for interacting with Gazebo from ROS2.
   </description>
 
-  <maintainer email="jrivero@osrfoundation.org">Jose Luis Rivero</maintainer>
+  <author>John Hsu</author>
+  <maintainer email="ecorbellini@ekumenlabs.com">Ernesto Corbellini</maintainer>
```

Comment:
Are the author and maintainer changes on purpose?

Reply:
Oops, will fix